client/interledger/js/components/interledger.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react/';
+
+import { Navbar, Row, Col, Button } from 'react-bootstrap/lib/';
+
+import { safeMerge } from 'js-utility-belt/es6';
+
+import AccountList from '../../../lib/js/react/components/accounts';
+// import Assets from './assets';
+
+import AssetActions from '../../../lib/js/react/actions/asset_actions';
+import AssetStore from '../../../lib/js/react/stores/asset_store';
+
+const Interledger = React.createClass({
+
+    getInitialState() {
+        const assetStore = AssetStore.getState();
+
+        return safeMerge(
+            {
+                activeAccount: null,
+                activeAsset: null,
+                activeLedger: null,
+                searchQuery: null
+            },
+            assetStore
+        );
+    },
+
+    componentDidMount() {
+        AssetStore.listen(this.onChange);
+        this.fetchAssetList();
+    },
+
+    componentWillUnmount() {
+        AssetStore.unlisten(this.onChange);
+        this.disconnectLedger(this.state.activeLedger);
+    },
+
+    onChange(state) {
+        this.setState(state);
+    },
+
+    fetchAssetList() {
+        AssetActions.flushAssetList();
+        const { activeAccount, searchQuery } = this.state;
+        const accountPublicKey = activeAccount ? activeAccount.vk : null;
+
+        AssetActions.fetchAssetList({ accountToFetch: accountPublicKey, search: searchQuery });
```

Comment:
Could be more concise by either renaming the local variables in this function or by renaming them in `fetchAssetList` so that you'd just have to do: `AssetActions.fetchAssetList({accountToFetch, search})`.